### DISCUSSION TITLE
Expecting "postal stop -h" to give a help message, but it stops the service

### DISCUSSION THREAD

**Discussion by arthurzenika** [postalserver/postal#2512] [Feature suggestions]
## Describe the bug

On a docker deployment of postal I wanted to restart only the web component, so I typed `postal stop -h` expecting to get some help to see if or how I could only stop one component. Instead I got a stop for all components. Nothing tragic, but 

`postal start -h` shows some docker-compose help... which is more failsafe. 

## To Reproduce

1. ssh into postal server
2. launch `postal stop -h`
3. see postal stopping. 

![image](https://github.com/postalserver/postal/assets/445200/ba29cf27-031f-443c-99a6-384be246fd42)

While I'm at it, the "Removing" part is necessary ? 

## Expected behaviour

Provide some help message. 

## Environment details 

 - OS: Linux debian
 - Browser n/a
 - Version 2.1.2
 - Type n/a

**Comment by willpower232** [comment]
You can see how that translates here https://github.com/postalserver/install/blob/ce3ddfa66e18a7509ebec9c7aebf84fcd70675df/bin/postal#L85-L91

Its a wrapper around docker compose which is why the removing part happens and also the help text you did see, if any, would probably have been confusing or inappropriate.

In this case the problem would be that the script doesn't error if there are too many arguments or the arguments are invalid. If you're familiar with bash then why not make a PR?
